data_example_test.go

```diff
@@ -208,3 +237,37 @@ func ExampleData_Compare_integers() {
 
 	// Output: [1 5 4]
 }
+
+func ExampleDataset_CopyNTimes() {
+	d1 := make(integers, 3)
+	d2 := integers{1, 2, 5}
+	d1.CopyNTimes(d2, 0, 0, []int{1, 1, 1})
+	fmt.Println(d1)
+
+	d1 = make(integers, 3)
+	d2 = integers{1, 2, 5}
+	d1.CopyNTimes(d2, 0, 0, []int{2, 1})
+	fmt.Println(d1)
+
+	d1 = make(integers, 3)
+	d2 = integers{1, 2, 5}
+	d1.CopyNTimes(d2, 0, 0, []int{1})
+	fmt.Println(d1)
+
+	d1 = make(integers, 3)
+	d2 = integers{1, 2, 5}
+	d1.CopyNTimes(d2, 1, 0, []int{1})
```

Comment:
lets demonstrate 0 case:
```suggestion
	d1.CopyNTimes(d2, 1, 0, []int{0, 2})
```